README.md

```diff
@@ -188,6 +188,26 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+### Conditionally including or excluding attributes
+
+You can also dynamically include or exclude serialized attributes by defining a
+method named include_attributes. This is typically used to customize the output
+based on `current_user`. For example:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :body, :author
+
+  def include_attributes(included)
+    if current_user.admin?
+      included
+    else
+      included - [:author]
+    end
+  end
```

Comment:
> Isn't this a lot of work just to be able to

> ```ruby
def attributes(*)
  super.tap do |attrs|
    # not sure where current_user is coming from.. 
    # is it an attribute on the object? 
    # an accessor on the serializer?
    attrs.delete(:author) unless current_user.admin? 
  end
```

The PR is different as it allows serializer to skip the computation of the attributes. Overriding `attributes` does not work when you're concerned about performance.